src/sendqueue.rs

```diff
@@ -19,7 +20,7 @@ impl SendQueue {
         let squeue = unsafe { raw::pcap_sendqueue_alloc(memsize) };
         let squeue = NonNull::new(squeue).ok_or(Error::InsufficientMemory)?;
 
-        Ok(Self(squeue))
+        Ok(Self(Handle::new(squeue, raw::pcap_sendqueue_destroy)))
```

Comment:
For consistency with `Capture` and `Savefile`, please implement a `From<NonNull<raw::pcap_send_queue>>` trait and use that.